client/stylesheets/_toCat.scss

```diff
@@ -4167,7 +4167,7 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 .profileCardUl > li:hover {
-    border: 1px solid #f8504b;
+    border: 1px solid #999;
```

Comment:
Color literals like `#999` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces